### PR TITLE
Contributing guidelines

### PR DESCRIPTION
**Pull request by simonekranawetter**
A markdown file for how to contribute with everything you need for set up and opening a pull request.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# Contributing
+
+Hey there! Thanks for thinking about helping us bring new features for Athena, the modern polymaths discord bot, to life. She will have more awesome features because you are helping contribute.
```
Comment (ThatNerdSquared) [inline CONTRIBUTING.md:3]: *The Modern Polymath's Discord bot ?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+Some setup is required in order to start contributing to Athena. Start by copying the `.env.example` file into a new file named `.env`.
+
+#### Node JS
+
+Athena is based on [NodeJS](https://nodejs.org/en/), which is required to run it. If you haven't already, you'll need to go ahead and download it. Athena is running on Node.js 16.9 and up.
+
+#### Installing Dependencies
+
+Now that you have Node installed, you can go ahead and install the required dependencies by running `npm ci` in the root directory _after installing NodeJS_. This may take a while. Have a coffee while you wait! []~(￣▽￣)~*
```
Comment (ThatNerdSquared) [inline CONTRIBUTING.md:43]: Probably want to put a `\` before the `~` here

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+- **Check out the FAQ** [below!](#faq)
+
+## Pull Request Guidelines <a name = "pullreqguide"></a>
+
+_Please adhere to the following guidelines._ We want to avoid duplicate issues, reports, features, etc.
+
+1. Search the [Issue Tracker](https://github.com/The-Modern-Polymath/athena/issues) for any relevant threads to what you're looking to create a PR for.
+2. Keep it detailed. We want to know your vision, and the more you give us the better discussion we can have!
+3. Please make sure you have read the [license agreement](LICENSE.md). We're using the [MIT License](https://opensource.org/licenses/MIT)- it's only 3 paragraphs!  
+4. Make a new branch to fork your changes from
```
Comment (ThatNerdSquared) [inline CONTRIBUTING.md:79]: "Fork the main repo and make a new branch for your changes" maybe?

**Review by ThatNerdSquared — CHANGES_REQUESTED** [review]
Looks good, just a few minor edits before the merge :)

File: CONTRIBUTING.md (hunk 3 above)
Comment (simonekranawetter) [reply CONTRIBUTING.md]: Done. Changed all 3 things and ready for another review. 😄

**Review by ThatNerdSquared — APPROVED** [review]
🥳

**Pull request merged**